flink-libraries/flink-table/src/main/scala/org/apache/flink/api/scala/table/expressionDsl.scala

```diff
@@ -223,10 +223,28 @@ trait ImplicitExpressionOperations {
     */
   def toTime = Cast(expr, SqlTimeTypeInfo.TIME)
 
-    /**
+  /**
     * Parses a timestamp String in the form "yy-mm-dd hh:mm:ss.fff" to a SQL Timestamp.
     */
   def toTimestamp = Cast(expr, SqlTimeTypeInfo.TIMESTAMP)
+
+  /**
+    * Accesses the field of a Flink composite type (such as Tuple, POJO, etc.) by name and
+    * returns it's value.
+    *
+    * @param name name of the field (similar to Flink's field expressions)
+    * @return value of the field
+    */
+  def getField(name: String) = GetCompositeField(expr, name)
```

Comment:
At the moment we specify the field name as a String anyway, so `'*` is not necessary.